### PR TITLE
allow second argument in login command

### PR DESCRIPTION
**Pull request by anvddriesch**
Towards https://github.com/giantswarm/roadmap/issues/1161

### CODE REVIEW

File: cmd/login/clientcert.go [hunk 1]
```diff
@@ -248,8 +248,8 @@ func getPrivKey(keyPEM []byte) (*rsa.PrivateKey, error) {
 	return x509.ParsePKCS1PrivateKey(block.Bytes)
 }
 
-// storeWCCredentials saves the created client certificate credentials into the kubectl config.
-func storeWCCredentials(k8sConfigAccess clientcmd.ConfigAccess, fs afero.Fs, c credentialConfig) (string, bool, error) {
+// storeWCClientCertCredentials saves the created client certificate credentials into the kubectl config.
```
Comment (anvddriesch) [inline cmd/login/clientcert.go:251]: Some renaming to show differences to client cert creation

File: cmd/login/login.go [hunk 2]
```diff
@@ -6,26 +6,20 @@ import (
...
-func (r *runner) handleMCLogin(ctx context.Context, installationIdentifier string) error {
-	if _, contextType := kubeconfig.IsKubeContext(installationIdentifier); contextType == kubeconfig.ContextTypeMC {
-		return r.loginWithKubeContextName(ctx, installationIdentifier)
+func (r *runner) findContext(ctx context.Context, installationIdentifier string) (bool, error) {
+	if _, contextType := kubeconfig.IsKubeContext(installationIdentifier); contextType != kubeconfig.ContextTypeNone {
+		return true, r.loginWithKubeContextName(ctx, installationIdentifier)
 
-	} else if kubeconfig.IsCodeName(installationIdentifier) {
-		return r.loginWithCodeName(ctx, installationIdentifier)
+	} else if kubeconfig.IsCodeName(installationIdentifier) || kubeconfig.IsWCCodeName(installationIdentifier) {
```
Comment (anvddriesch) [inline cmd/login/login.go:18]: We allow logging in with a WC context name

**Comment by anvddriesch** [comment]
I will open a second PR for renaming clientcert contexts with `-clientcert` suffix. We can not merge this without this other PR.

**Review by kuosandys — APPROVED** [review]
I had some issues testing out the whole WC OIDC login flow, but allowing a second argument for the command works well from what I can test, and the fallback to `-clientcert` context works as expected 🙂

**Pull request merged**